tendrl/commons/utils/alert_util.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+import socket
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils.time_utils import now as tendrl_now
+
+
+def alert_job_status(curr_value, msg):
+    alert = {}
+    alert['source'] = NS.publisher_id
+    alert['pid'] = os.getpid()
+    alert['time_stamp'] = tendrl_now().isoformat()
+    alert['alert_type'] = 'status'
+    severity = "INFO"
+    if curr_value.lower() == "failed":
+        severity = "CRITICAL"
+    alert['severity'] = severity
+    alert['resource'] = 'cluster'
+    alert['current_value'] = curr_value
+    alert['tags'] = dict(
+        message=msg,
+        cluster_id=NS.tendrl_context.integration_id,
+        cluster_name=NS.tendrl_context.cluster_name,
+        sds_name=NS.tendrl_context.sds_name,
+        fqdn=socket.getfqdn()
+    )
+    alert['node_id'] = NS.node_context.node_id
+    if not NS.node_context.node_id:
+        return
+    Event(
```

Comment:
use logger utils

Reply:
Ok. Will check and use that

Reply:
I am not sure log_utils could be used for `notice` type. Still would verify once with @GowthamShanmugam and @rishubhjain on this.